apps/cowswap-frontend/src/legacy/hooks/useIsAmbireWallet.ts

```diff
@@ -1,7 +1,6 @@
-import { useWalletMetaData } from 'modules/wallet'
-import { getIsAmbireWallet } from 'modules/wallet/api/utils/connection'
+import { useWalletMetaData, getIsAmbireWallet } from 'modules/wallet'
 
-export default function useIsAmbireWallet(): boolean {
```

Comment:
this hook uses only stuff from the wallet module, so I wonder if this is where it belongs. 

Some additional idea, I think this is only used for detecting if the wallet is a smart contract wallwet. I think this should also be part of the wallet module. This way, we don't need to see any "ambire" use outside of the module

Reply:
I've moved the hook into `useIsSmartContractWallet` here:
https://github.com/cowprotocol/cowswap/pull/3109